pkgs.nix

```diff
@@ -135,4 +137,6 @@ let
     clockify = taxiClockify;
   };
 in
-taxi
+{
+  inherit taxi withPlugins availablePlugins;
```

Comment:
Why is this needed?  `withPlugins` and `availablePlugins` can be accessed through `taxi.withPlugins` and `taxi.availablePlugins`.

Reply:
Not needed anymore, thanks. Just removed it